ruby/files_and_serialization/project_event_manager.md

```diff
@@ -4,12 +4,12 @@ Please note this tutorial has been adapted from The Turing School's and Jump Sta
 
 After completing this tutorial, you will be able to:
 
-* manipulate [file](http://rubydoc.info/stdlib/core/File) input and output
+* manipulate [file](http://rubydoc.info/stdlib/core-3.1.2/File) input and output
```

Comment:
This link doesn't work for me, I think version agnostic links should generally be fine to leave